### PR TITLE
Root classes cleanup

### PR DESCRIPTION
**Pull request by Valodim**
More cleanup, particularly of mFieldNames. This is almost 100% android studio refactoring, and should contain no semantic changes whatsoever.

### CODE REVIEW

File: k9mail/src/main/java/com/fsck/k9/EmailAddressValidator.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 public class EmailAddressValidator implements Validator {
     private static final Pattern EMAIL_ADDRESS_PATTERN = Pattern.compile(
-          "[a-zA-Z0-9\\+\\.\\_\\%\\-\\+]{1,256}" +
+          "[a-zA-Z0-9\\+\\.\\_\\%\\-]{1,256}" +
```
Comment (philipwhiuk) [inline k9mail/src/main/java/com/fsck/k9/EmailAddressValidator.java]: We're already matching +. Not sure why this changed.

Comment (Valodim) [inline k9mail/src/main/java/com/fsck/k9/EmailAddressValidator.java]: Precisely because we are already matching for the +. This is a character class

Comment (philipwhiuk) [inline k9mail/src/main/java/com/fsck/k9/EmailAddressValidator.java]: Oh, I think I was sleep deprived. You're removing the duplicate.

**Comment by cketti** [comment]
:+1:

**Pull request merged**